packages/runtime-dom/__tests__/apiCustomElement.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { parseNumber } from '../src/apiCustomElement'
```

Comment:
The test file `customElement.spec.ts` already exists 😂

Reply:
Cmd + P betrayed me! 😆